payjoin/src/receive/v2/mod.rs

```diff
@@ -56,6 +56,11 @@ impl SessionContext {
             .join(&format!("/{directory_base}"))
             .map_err(|e| InternalSessionError::ParseUrl(e.into()))
     }
+
+    /// The per-session identifier
+    pub fn id(&self) -> ShortId {
```

Comment:
One thing did just come to mind. Does this need to be pub? IIRC only internal types need access to this?
Seems like it was pub previously as well.

Reply:
should be `pub(crate)` because it's on SessionContext and not `receiver.id()`